.claude/epics/phase-4-intelligence/github-mapping.md

```
@@ -0,0 +1,55 @@
+# GitHub Issue Mapping
+
+Epic: #3 - https://github.com/curdriceaurora/Local-File-Organizer/issues/3
+
+## Tasks:
+
+### File Deduplication
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)